rr_platform/launch/camera_calibration_18064461.yaml

```diff
@@ -0,0 +1,20 @@
+image_width: 1280
+image_height: 964
+camera_name: 18064461
+camera_matrix:
+  rows: 3
+  cols: 3
+  data: [730.089675, 0.000000, 662.572253, 0.000000, 734.015999, 486.144443, 0.000000, 0.000000, 1.000000]
+distortion_model: plumb_bob
+distortion_coefficients:
+  rows: 1
+  cols: 5
+  data: [-0.247334, 0.049363, 0.000464, 0.000510, 0.000000]
+rectification_matrix:
+  rows: 3
+  cols: 3
+  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
```

Comment:
This is fine, I don't think we need to worry about rectification for this summer

Reply:
The rectification matrix just has to do with stereo camera pairs.
The distortion coefficients, camera matrix, and projection matrix work together to rectify monocular systems.

Reply:
So it's actually doing proper rectification?

Reply:
Yeah
`/pointgrey_camera/image_color_rect` will give you an undistorted image (the whole, real straight lines look straight bit).

Reply:
Nice